.github/workflows/on-doc-merge.yml

```diff
@@ -0,0 +1,21 @@
+name: Docs
+
+on:
+  push:
+    branches:
```

Comment:
I think this should run on PR too

Reply:
What we _might_ want to do on merge to main, is just build the documentation, to make sure we still can.

Reply:
Hey @scottgerring  - I've updated configuration. Please have a look.